README.md

```diff
@@ -22,3 +28,17 @@ version = 375
  status = new
   names = fault-tolerant-execution-task-memory-growth-factor, optimizer.non-estimatable-predicate-approximation.enabled
 ```
+
+### Update the property index 
+
+In case that the local properties index of the project does not include the properties corresponding to the latest Trino release,
```

Comment:
```suggestion
In case that the local properties index of the project does not include the properties corresponding to a particular Trino release,
```